fixed 80 char length or more lines

zouhir opened the pull request.

zouhir committed:
fixed 80 char length or more lines

zouhir committed:
fixed conflict

Mikumiku747 commented on base64.c:
```diff
@@ -24,7 +24,9 @@
  * 64 character table for converting to Base64
  */
-char b64_table[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+char b64_table[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZ\
```
The way you've split the strings here will break the program, since that backslash only escapes the newline character, and not the rest of it. I'll give it a quick fix and it should be fine. 

The way to split strings up into C is to separate them into separate strings, like so:
```
char longString[] = "This is the first part of the string, but it's very long so "
        "I can split them up like so, and the compiler will realise that since there's "
        "nothing between them, they must all be part of a single string.";
```

As it is now, they all have a bunch of tab characters in them, which will throw off the base64 encoder.

Mikumiku747 committed:
Fixed string splitting

They were split in a broken way, but they're good now. Uses the fact that adjacent strings in C are treated as a single string, it's specifally for this purpose I think. C is wierd sometimes.

Mikumiku747 reviewed (commented):
I made a quick change to the string splitting so it wasn't broken, let me know if that's OK.

zouhir commented:
Looks great
